sources/api/schnauzer/src/helpers.rs

```diff
@@ -1346,6 +1356,138 @@ handlebars_helper!(any_enabled: |arg: Value| {
     result
 });
 
+/// This helper writes out the default OCI runtime spec.
+///
+/// Default settings are specified in the following files:
+/// * sources/models/shared-defaults/oci-resource-limits.toml
+/// * sources/models/shared-defaults/oci-capabilities.toml
+
+fn get_param_path<'a>(helper: &'a Helper<'_, '_>, index: usize) -> Result<&'a String, RenderError> {
+    Ok(helper
+        .params()
+        .get(index)
+        .context(error::MissingParamSnafu {
+            index,
+            helper_name: helper.name(),
+        })?
+        .relative_path()
+        .context(error::MissingParamPathSnafu {
+            index,
+            helper_name: helper.name(),
+        })?)
+}
+
+pub fn oci_defaults(
```

Comment:
describe how helper is called, that it gets setting names from the blob itself, and that there are specific setting names it expects, or else it will error